### ISSUE TITLE
Project Feedback

### ISSUE THREAD

**Issue by wolyniec** [wolyniec/MI-449-SS17-741-html-atom-and-basic-elements-JmL5o8#1]
@stuartpearman Can you take a look at this? It's [hosted here](https://wolyniec.github.io/MI-449-SS17-741-html-atom-and-basic-elements-JmL5o8/) and meets the following criteria:

- [x] Each of the following elements is used appropriately, at least once: `h1`, `h2`, `p`, `img`, `li`, `a`
- [x] At least one of the following elements is used: `ol`, `ul`
- [x] At least one external image is used and links to the page it came from

**Comment by stuartpearman** [comment]
Hey Ivan, this is a good start. A couple things:

### `br` tags

`br` tags should be reserved for times when a line break fits the content specifically, like an address or a poem. If you want to make a paragraph less wide, you should generally use CSS for this. For now, I wouldn't worry about it, we'll cover layout and CSS in a later lesson.

### `hr` tag

Try replacing the `~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-` with an `hr` tag. This element is used to create lines that break up the content.

### Nesting in paragraphs

In general, you want to avoid nesting headings, lists, and other block level elements in your paragraph tags. These are separate content items and they should begin after the end of your paragraph, etc.

**Comment by wolyniec** [comment]
Right on! I appreciate the feedback and have incorporated it into a new build of the project. What do you think?

**Comment by stuartpearman** [comment]
Yeah, this is better! The only thing I would do now is wrap any floating text in a paragraph tag, so the first paragraph describing Kurt's paranormal background, and the last bits at the end.

Let me know if you need any guidance with this.

**Comment by wolyniec** [comment]
Alright I should have fixed the basic formatting of this (ancient) project

**Comment by stuartpearman** [comment]
Nice work 🙂  :shipit:
